packages/jsii-rosetta/lib/snippet.ts

```diff
@@ -30,8 +30,22 @@ export interface TypeScriptSnippet {
    * @default false
    */
   readonly strict?: boolean;
+
+  /**
+   * Dependencies necessary to compile this snippet
+   *
+   * Value is a regular { name -> semver } map like NPM's `dependencies`,
```

Comment:
Is that true, though? CompilationDependency is not `semver` :)

Reply:
Well. The string that's there is supposed to be :).